yeat/tests/test_paired.py

```diff
@@ -7,83 +7,108 @@
+        wd,
+        data_file(f"configs/{algorithm}.cfg"),
+    ]
+    args = cli.get_parser().parse_args(arglist)
+    cli.main(args)
+    observed = Path(wd).resolve() / "analysis" / "sample1" / label / algorithm / expected
+    assert observed.exists()
 
 
 @pytest.mark.long
-def test_multiple_assemblers(capsys, tmp_path):
+def test_multiple_spades(capsys, tmp_path):
     wd = str(tmp_path)
     arglist = [
         "--outdir",
         wd,
-        "--paired",
-        data_file("short_reads_1.fastq.gz"),
-        data_file("short_reads_2.fastq.gz"),
-        data_file("config.cfg"),
+        data_file(f"configs/two_spades.cfg"),
     ]
     args = cli.get_parser().parse_args(arglist)
     cli.main(args)
-    spades_result = Path(wd).resolve() / "analysis" / "spades" / "contigs.fasta"
-    assert spades_result.exists()
-    megahit_result = Path(wd).resolve() / "analysis" / "megahit" / "final.contigs.fa"
-    assert megahit_result.exists()
+    analysis_dir = Path(wd).resolve() / "analysis"
+    expected = [
+        analysis_dir / "sample1" / "spades-default" / "spades" / "contigs.fasta",
+        analysis_dir / "sample1" / "spades-meta" / "spades" / "contigs.fasta",
+    ]
+    for observed in expected:
+        assert observed.exists()
 
 
 @pytest.mark.long
-def test_unicycler(capsys, tmp_path):
+def test_multiple_paired_end_assemblers(capsys, tmp_path):
     wd = str(tmp_path)
     arglist = [
         "--outdir",
         wd,
-        "--paired",
-        data_file("short_reads_1.fastq.gz"),
-        data_file("short_reads_2.fastq.gz"),
-        data_file("unicycler.cfg"),
+        data_file(f"configs/unicycler_spades.cfg"),
     ]
     args = cli.get_parser().parse_args(arglist)
     cli.main(args)
-    assembly_result = Path(wd).resolve() / "analysis" / "unicycler" / "assembly.fasta"
-    assert assembly_result.exists()
+    analysis_dir = Path(wd).resolve() / "analysis"
+    expected = [
+        analysis_dir / "sample1" / "unicycler-default" / "unicycler" / "assembly.fasta",
+        analysis_dir / "sample1" / "spades-meta" / "spades" / "contigs.fasta",
+    ]
+    for observed in expected:
+        assert observed.exists()
+
+
+# @pytest.mark.long
+# def test_run_complex_paired_end_config(capsys, tmp_path):
+#     wd = str(tmp_path)
+#     arglist = [
+#         "--outdir",
+#         wd,
+#         data_file("configs/paired.cfg"),
+#     ]
+#     args = cli.get_parser().parse_args(arglist)
+#     cli.main(args)
+#     analysis_dir = Path(wd).resolve() / "analysis"
+#     expected = [
+#         analysis_dir / "sample1" / "default_spades" / "spades" / "contigs.fasta",
+#         analysis_dir / "sample2" / "default_spades" / "spades" / "contigs.fasta",
+#         analysis_dir / "sample2" / "default_megahit" / "megahit" / "final.contigs.fa",
+#         analysis_dir / "sample1" / "Shigella_sonnei_plasmids" / "unicycler" / "assembly.fasta",
+#     ]
+#     for observed in expected:
+#         assert observed.exists()
```

Comment:
This test is extremely long--takes almost an hour. It runs 4 assemblers with two different data sets. The quickest one is `short_reads_*.fastq.gz` and the longest one is `Animal_289_R*.fq.gz`. I'm going to remove this test and the short read dataset of `Animal_289_R*.fq.gz` because they take too long to assemble for testing purposes.